### ISSUE TITLE
Telegraf 0.1.5 sample config not functional

### ISSUE THREAD

**Issue by srfraser** [influxdb/telegraf#121]
Installing from telegraf_0.1.5_amd64.deb and running `telegraf -sample-config` the output contains the old-style 

```
# Configuration for influxdb server to send metrics to
[influxdb]
# The full HTTP endpoint URL for your InfluxDB instance
url = "http://localhost:8086" # required.

# The target database for metrics. This database must already exist
database = "telegraf" # required.
```

README.md contains the required configuration format:
```
[outputs]
[outputs.influxdb]
url = "http://192.168.59.103:8086" # required.
database = "telegraf" # required.
```

**Comment by srfraser** [comment]
Ok, so if you don't have the `outputs` section then the log file contains:

```
2015/08/19 11:04:09 Error: no outputs found, did you provide a valid config file?
```

But `telegraf -config /etc/opt/telegraf/telegraf.conf -test` works, and produces output. 

If you specify the `outputs` section then the `-test` option no longer works:
```
2015/08/19 11:06:07 Undefined but requested plugin: outputs
```

**Comment by sparrc** [comment]
@srfraser did you download the new 0.1.5 deb? (uploaded within the last 24 hours)

**Comment by srfraser** [comment]
This is from a .deb I downloaded about 8 hours ago

**Comment by sparrc** [comment]
Okay, thanks for the report, looking into it

**Comment by sparrc** [comment]
This should now be fixed, please download the 0.1.5 debs again and re-open if needed
